test/unit/liquidity_pool_endpoints_test.js

```diff
@@ -141,4 +141,492 @@ describe('/liquidity_pools tests', function() {
+              }
+            },
+            "id": "2ff47e1bafe68639276b2a8df0a73597ee0c062fbcc72d121af314fe7851c908",
+            "paging_token": "3847380164161536",
+            "successful": true,
+            "hash": "2ff47e1bafe68639276b2a8df0a73597ee0c062fbcc72d121af314fe7851c908",
+            "ledger": 895788,
+            "created_at": "2021-08-09T20:53:11Z",
+            "source_account": "GAHQN6YNYD6ZT7TLAVE4R36MSZWQJZ22XB3WD4RLSHURMXHW4VHJIDF7",
+            "source_account_sequence": "3847371574214658",
+            "fee_account": "GAHQN6YNYD6ZT7TLAVE4R36MSZWQJZ22XB3WD4RLSHURMXHW4VHJIDF7",
+            "fee_charged": "10000",
+            "max_fee": "10001",
+            "operation_count": 1,
+            "envelope_xdr": "AAAAAgAAAAAPBvsNwP2Z/msFScjvzJZtBOdauHdh8iuR6RZc9uVOlAAAJxEADasqAAAAAgAAAAAAAAAAAAAAAQAAAAEAAAAADwb7DcD9mf5rBUnI78yWbQTnWrh3YfIrkekWXPblTpQAAAAGAAAAAVNFQwAAAAAAm6XFaVsf8OSuS9C9gMplyTjagE9jAnnqwxSDJ6fin6IAsaK8LsUAAAAAAAAAAAAB9uVOlAAAAECXmRsoXmRiJjUrtbkDZYRnzac5s1CVV4g2RlIgBIuQty21npz3A1VhUcSmAx+GmsyGxVFvIrcdstTawJlmy9kF",
+            "result_xdr": "AAAAAAAAJxAAAAAAAAAAAQAAAAAAAAAGAAAAAAAAAAA=",
+            "result_meta_xdr": "AAAAAgAAAAIAAAADAA2rLAAAAAAAAAAADwb7DcD9mf5rBUnI78yWbQTnWrh3YfIrkekWXPblTpQAAAAAGtJNDAANqyoAAAABAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAAAAAABAA2rLAAAAAAAAAAADwb7DcD9mf5rBUnI78yWbQTnWrh3YfIrkekWXPblTpQAAAAAGtJNDAANqyoAAAACAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAAAAAABAAAAAwAAAAMADassAAAAAAAAAAAPBvsNwP2Z/msFScjvzJZtBOdauHdh8iuR6RZc9uVOlAAAAAAa0k0MAA2rKgAAAAIAAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAAAAAEADassAAAAAAAAAAAPBvsNwP2Z/msFScjvzJZtBOdauHdh8iuR6RZc9uVOlAAAAAAa0k0MAA2rKgAAAAIAAAABAAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAAAAAAADassAAAAAQAAAAAPBvsNwP2Z/msFScjvzJZtBOdauHdh8iuR6RZc9uVOlAAAAAFTRUMAAAAAAJulxWlbH/DkrkvQvYDKZck42oBPYwJ56sMUgyen4p+iAAAAAAAAAAAAsaK8LsUAAAAAAAEAAAAAAAAAAAAAAAA=",
+            "fee_meta_xdr": "AAAAAgAAAAMADasrAAAAAAAAAAAPBvsNwP2Z/msFScjvzJZtBOdauHdh8iuR6RZc9uVOlAAAAAAa0nQcAA2rKgAAAAEAAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAAAAAEADassAAAAAAAAAAAPBvsNwP2Z/msFScjvzJZtBOdauHdh8iuR6RZc9uVOlAAAAAAa0k0MAA2rKgAAAAEAAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAA==",
+            "memo_type": "none",
+            "signatures": [
+              "l5kbKF5kYiY1K7W5A2WEZ82nObNQlVeINkZSIASLkLcttZ6c9wNVYVHEpgMfhprMhsVRbyK3HbLU2sCZZsvZBQ=="
+            ]
+          }
+        ]
+      }
+    };
+
+    it('retrieves its transactions', function(done) {
+      this.axiosMock
+        .expects('get')
+        .withArgs(sinon.match(`${LP_URL}/${lpId}/transactions`))
+        .returns(Promise.resolve({ data: poolTxsResponse }))
+
+      this.server
+        .transactions()
+        .forLiquidityPool(lpId)
+        .call()
+        .then((poolTxs) => {
+          expect(poolTxs.records).to.deep.equal(poolTxsResponse._embedded.records);
+          done();
+        })
+        .catch(done);
+    });
+
+    const poolFxsResponse = {
```

Comment:
Is `Fxs` an acronym for effects? It would be clearer to have a const called `poolEffectsResponse` here since Fxs doesn't seem to be used anywhere else in the project.

Reply:
Yeah, it was supposed to parallel `Txs` but that's clearly confusing haha, fixed.